### ISSUE TITLE
Results questions

### ISSUE THREAD

**Issue by laurafdeza** [laurafdeza/Dissertation#14]
@jvcasillas, the draft for the results of the experiment we were talking about the other day on L1 transfer and L2 proficiency on prediction are [here](https://github.com/laurafdeza/Dissertation/blob/master/reports/stress_use.Rmd).
Some questions:
1. When should ß and χ be included or referred to?
2. In the plots, how can I do so that the effects of proficiency and L2 use are shown in different colors depending onthe condition? Now the line is different for each tense but the dots are all blue. The closest I got was laying over two plots but the gradient did not show for one of the tenses.
3. GLMMs should also be in Appendices?
4. In the report there are some hidden comments about how much to interpret the results. I feel that sometimes I may be interpreting too much.

The GCAs are [here](https://github.com/laurafdeza/Dissertation/blob/master/scripts/stress/analyses/03_analysis_gca_use_NEW.R), their plots [here](https://github.com/laurafdeza/Dissertation/blob/master/scripts/stress/analyses/06_plots_gca_use_NEW.R) and the GLMMs [here](https://github.com/laurafdeza/Dissertation/blob/master/scripts/stress/analyses/02_analysis_glmm_use.R) (this script also includes some attempts to plot the GLMMs, although not always completely successfully).

**Comment by jvcasillas** [comment]
Estoy con el teléfono ahora así que no puedo responder a todas las preguntas pero comento lo que me resulta cómodo ahora y vuelvo más tarde. ￼

Usa χ cuando reportas cualquier efecto principal (o interacción) que hayas testado usando nested model comparisons. Reportas β si se trata de un parameter estimate. 

Para ayudarte con 2 es más fácil si me pones un reprex o si copias tu código aquí (github formatea el código bien y es fácil de ver, usa backticks como en los knitr code chunks). 

No entiendo 3. Todos los glmms? Tablas?

**Comment by jvcasillas** [comment]
4 no es una pregunta pero miraré tus ejemplos y te digo.

**Comment by laurafdeza** [comment]
¡Claro, cuando puedas!

¿Y cuál es entonces la diferencia entre ß y γ? Tomé como referencia para escribir los resultados el artículo que tienes con Nuria del 2018 y el de Cristina de los intérpretes, y en el primero usáis ß y en el de Cristina gammas.

Sobre los GLMMs, sí las tablas con los fixed and random effects.

Para la pregunta 2, un código como este:
`model_preds$fits_all_l2_use %>%
  mutate(condition = if_else(condition_sum == 1, "Present", "Preterit"),
         condition = fct_relevel(condition, "Present"),
         l1 = if_else(l1 == 'en', 'English', 'Mandarin'),
         l1 = fct_relevel(l1, "English", "Mandarin"))%>%
  ggplot(., aes(x = time_zero, y = fit, ymax = ymax, ymin = ymin,
                fill = condition, color = percent_l2_week)) +
  facet_wrap(l1 ~ .) +
  geom_hline(yintercept = 0, lty = 3, size = 0.4) +
  geom_vline(xintercept = 4, lty = 3, size = 0.4) +
  stat_summary(fun.y = "mean", geom = "line", size = 1) + 
  
  stat_summary(fun.data = mean_cl_boot, geom = 'ribbon',fun.args=list(conf.int=0.95),
               alpha = 0.5) +
  geom_point(aes(color = percent_l2_week), size = 1.3, show.legend = F) +
  geom_point(aes(color = percent_l2_week), size = 0.85, show.legend = F) +
  scale_x_continuous(breaks = c(-4, 0, 4, 8, 12),
                     labels = c("-200", "0", "200", "400", "600")) +
  labs(x = "Time (ms) relative to target syllable offset",
       y = "Empirical logit of looks to target") +
  scale_fill_brewer(palette = 'Set1', name = "Tense",
                    labels = c("Present", "Preterit")) +
  theme_big + labs(color = "Weekly L2 % use") +
  theme(
    legend.position = c(0.1, 0.75),
    legend.key = element_blank(),
    legend.background = element_blank(),
    strip.background = element_blank(),
    axis.title.y = element_text(size = rel(.9), hjust = 0.95),
    axis.title.x = element_text(size = rel(.9)),
    legend.key.size = unit(0.75, 'lines'),
    legend.text = element_text(size = 6),
    legend.title = element_text(size = 7),
    plot.margin = unit(rep(2, 4), "mm"),
    panel.grid.major = element_line(colour = 'grey90', size = 0.15),
    panel.grid.minor = element_line(colour = 'grey90', size = 0.15)
  )
`
Da un gráfico como el que va adjuntado. Si te fijas, las líneas sí reflejan la condición de stress pero los puntos de L2 use no.
![l2use_tense_withingroup](https://user-images.githubusercontent.com/38184931/110790184-b919ef80-823e-11eb-869a-354662d01ef1.png)

**Comment by laurafdeza** [comment]
Y otra pregunta, debería normalizar proficiency y L2 use de alguna manera? Con z-scores o algo así. Pero si hago ese cambio, luego será más difícil interpretar los resultados, no?

**Comment by jvcasillas** [comment]
> ¡Claro, cuando puedas!
> 
> ¿Y cuál es entonces la diferencia entre ß y γ? Tomé como referencia para escribir los resultados el artículo que tienes con Nuria del 2018 y el de Cristina de los intérpretes, y en el primero usáis ß y en el de Cristina gammas.
> 

β se usa para referirse al parameter estimate (el valor de la estimación) y ɣ (en esa tabla) es un índice para vincular el parameter que se estima con los time terms. El primer subíndice indica si es un efecto sobre el intercept (0), el linear time term (1), el time term cuadrático (2) o el time term cúbico (3). El segundo subíndice agrupa los efectos (0 = intercept; 1 = syllable structure; 2 = lexical stress; etc.). Entonces, ɣ23 indica cuadratic efect on lexical stress. 

> Sobre los GLMMs, sí las tablas con los fixed and random effects.
> 

Si caben, deben estar en la parte principal del texto. 

> Para la pregunta 2, un código como este:

```
model_preds$fits_all_l2_use %>%
  mutate(condition = if_else(condition_sum == 1, "Present", "Preterit"), condition = fct_relevel(condition, "Present"), l1 = if_else(l1 == 'en', 'English', 'Mandarin'),
l1 = fct_relevel(l1, "English", "Mandarin"))%>%
ggplot(., aes(x = time_zero, y = fit, ymax = ymax, ymin = ymin,
fill = condition, color = percent_l2_week)) +
facet_wrap(l1 ~ .) +
geom_hline(yintercept = 0, lty = 3, size = 0.4) +
geom_vline(xintercept = 4, lty = 3, size = 0.4) +
stat_summary(fun.y = "mean", geom = "line", size = 1) +
stat_summary(fun.data = mean_cl_boot, geom = 'ribbon',fun.args=list(conf.int=0.95), alpha = 0.5) +
geom_point(aes(color = percent_l2_week), size = 1.3, show.legend = F) +
geom_point(aes(color = percent_l2_week), size = 0.85, show.legend = F) +
scale_x_continuous(breaks = c(-4, 0, 4, 8, 12),
labels = c("-200", "0", "200", "400", "600")) +
labs(x = "Time (ms) relative to target syllable offset", y = "Empirical logit of looks to target") +
scale_fill_brewer(palette = 'Set1', name = "Tense", labels = c("Present", "Preterit")) +
theme_big + labs(color = "Weekly L2 % use") +
theme(
  legend.position = c(0.1, 0.75),
  legend.key = element_blank(),
  legend.background = element_blank(),
  strip.background = element_blank(),
  axis.title.y = element_text(size = rel(.9), hjust = 0.95),
  axis.title.x = element_text(size = rel(.9)),
  legend.key.size = unit(0.75, 'lines'),
  legend.text = element_text(size = 6),
  legend.title = element_text(size = 7),
  plot.margin = unit(rep(2, 4), "mm"),
  panel.grid.major = element_line(colour = 'grey90', size = 0.15),
  panel.grid.minor = element_line(colour = 'grey90', size = 0.15))
```

> Da un gráfico como el que va adjuntado. Si te fijas, las líneas sí reflejan la condición de stress pero los puntos de L2 use no.
> ![l2use_tense_withingroup](https://user-images.githubusercontent.com/38184931/110790184-b919ef80-823e-11eb-869a-354662d01ef1.png)

Creo que aún no entiendo lo que quieres como resultado. Que los puntos tengan los mismos colores que las líneas? Recuerda que L2 use es una variable continua y stress es binaria. Descríbeme exactamente lo que quieres.

**Comment by jvcasillas** [comment]
> Y otra pregunta, debería normalizar proficiency y L2 use de alguna manera? Con z-scores o algo así. Pero si hago ese cambio, luego será más difícil interpretar los resultados, no?

Sí, con z-scores. No creo que sea más difícil, pero siempre puedes transformar todo de nuevo después, si quieres.

**Comment by laurafdeza** [comment]
Sí, querría que L2 proficiency variara de claro a oscuro según el color de la condición de stress. Algo como en la foto adjunta.
Probé también creando una interacción entre proficiency y stress pero seguía saliendo de un solo color, aunque obviamente pude haberlo hecho mal.
![PXL_20210313_171456443](https://user-images.githubusercontent.com/38184931/111038297-32a80e00-83f6-11eb-8f1c-4e519556292e.jpg)

**Comment by jvcasillas** [comment]
> Sí, querría que L2 proficiency variara de claro a oscuro según el color de la condición de stress. Algo como en la foto adjunta.
> Probé también creando una interacción entre proficiency y stress pero seguía saliendo de un solo color, aunque obviamente pude haberlo hecho mal.
> ![PXL_20210313_171456443](https://user-images.githubusercontent.com/38184931/111038297-32a80e00-83f6-11eb-8f1c-4e519556292e.jpg)

Ah ok. Ahora entiendo bien. Creo que vas a tener problemas porque en principio no se debe asignar el mismo aesthetic a dos variables diferentes, pero quizá puedas conseguir algo parecido a lo que quieres asignando alpha a una variable. 

```
mtcars %>% 
  mutate(vs = as.factor(vs)) %>% 
  ggplot(., aes(x = drat, y = mpg)) + 
    geom_point(aes(fill = vs, alpha = disp), pch = 21, color = "black", size = 2) + 
    geom_smooth(aes(color = vs), method = lm, se = F) + 
    scale_fill_manual(values = c("red", "blue")) + 
    scale_color_manual(values = c("red", "blue")) 
```

**Comment by laurafdeza** [comment]
Sí funciona, gracias! La verdad no estoy segura de que sea mucho más informativa.
Por lo demás, he actualizado el report con los datos de los z-scores. No había gran cambio.
![alpha_graph](https://user-images.githubusercontent.com/38184931/111039883-70109980-83fe-11eb-8656-aa6f2c02c0f4.png)

**Comment by jvcasillas** [comment]
Eso tiene sentido. Hacer una transformación de x score no cambia los resultados pero tiene ventajas computacionales (y como ya tarda mil años el gca...)

**Comment by laurafdeza** [comment]
Sí, es verdad que con z-scores tardaron menos.

**Comment by jvcasillas** [comment]
Comments added via https://github.com/laurafdeza/Dissertation/pull/15

**Comment by laurafdeza** [comment]
Recibido. Gracias!
